### ISSUE TITLE
Beer Song - Remove stub solution

### ISSUE THREAD

**Issue by lpalma** [exercism/xhaskell#510]
I'm not sure why Beer Song [has a stub solution](https://github.com/exercism/xhaskell/blob/master/exercises/beer-song/src/Beer.hs), which is basically a copy of the test, making it pass.

Shouldn't it follow the standard `error "You need to implement this function."` found in other exercises? The test and the README already provide the lyrics, so I don't think it is necessary to also have it in the `src` code.

**Comment by petertseng** [comment]
> I'm not sure why Beer Song has a stub solution

be3cause https://github.com/exercism/xhaskell/blob/master/exercises/beer-song/HINTS.md and #301

let us know if you disagree

**Comment by lpalma** [comment]
>because https://github.com/exercism/xhaskell/blob/master/exercises/beer-song/HINTS.md and #301 

oh... my bad. I should have paid more attention in the README. It makes sense now.

Actually this brought me another question, and I hope I'm not being pedantic here 😛 

Not all exercises have **Hints**, but all of them have the basic instruction sections _Getting Started_, _Running the tests_, etc. right after the problem description (or _For bonus points_, when available).

Shouldn't **Hints** be displayed before these instructions sections and after the problem description? Most of the students (or is it just me? 😕 ) would read those instruction sections only once and ignore them on the following exercises, not realising there maybe be more important information after that.

In other words, instruction sections are static/invariant, thus shouldn't be followed by anything.

Is it a question that should be asked in [x-common](https://github.com/exercism/x-common) instead?

**Comment by petertseng** [comment]
> Shouldn't Hints be displayed before these instructions sections and after the problem description? 
> Is it a question that should be asked in x-common instead?

Probably, since it relates to more tracks than just Haskell.

The change will need to be made in https://github.com/exercism/trackler/blob/master/lib/trackler/implementation.rb , though opening the issue in trackler repository will probably not reach enough of an audience.

**Comment by petertseng** [comment]
> In other words, instruction sections are static/invariant, thus shouldn't be followed by anything.

This makes sense to me. Right now, the order is:

* part specific to each exercise
* part not specific to each exercise
* part specific to each exercise

**Comment by lpalma** [comment]
>The change will need to be made in https://github.com/exercism/trackler/blob/master/lib/trackler/implementation.rb

thanks @petertseng 
It's good to know where things belong / how they are glued together :-)

I have opened the [Issue in Trackler](https://github.com/exercism/trackler/issues/26). Hopefully feedback will be given.
